src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperCloneCheck.java

```diff
@@ -37,7 +37,48 @@
  * <pre>
  * &lt;module name=&quot;SuperClone&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Foo {
+ *  protected int f;
+ *
+ *  public Foo() {};
+ *  public Foo(int f) {
+ *     this.f = f;
+ *  }
+ *  // violation, does not call super.clone()
+ *  public Object clone() {
```

Comment:
Why can't comment go on end of line? Violation is on the method, not before or after it.